lucet-runtime/lucet-runtime-tests/src/timeout.rs

```diff
@@ -215,7 +216,7 @@ macro_rules! timeout_tests {
             }
 
             // An instance that has faulted is not terminable.
-            assert_eq!(kill_switch.terminate(), Err(KillError::NotTerminable));
```

Comment:
The comment here says `not terminable`, and in a way the `Err` _does_ mean `not terminable`, but `Invalid` means the underlying `KillState` is gone and the `KillSwitch` is no longer valid, which is slightly different from the `NotTerminable` result. I think this is a case of the test being subtly incorrect, but I know we've talked a bit about this granularity of `KillError` maybe being a misdesigned interface anyway.